pkg/node/node.go

```diff
@@ -296,6 +299,8 @@ func (s *CSINodeService) NodeUnstageVolume(ctx context.Context, req *csi.NodeUns
 	if volumeCR.Annotations[fakeAttachVolumeAnnotation] == fakeAttachVolumeKey {
 		ll.Warningf("Removing fake-attach annotation for volume %s", volumeID)
 		delete(volumeCR.Annotations, fakeAttachVolumeAnnotation)
+		s.VolumeManager.recorder.Eventf(volumeCR, eventing.NormalType, eventing.FakeAttachDisabled,
```

Comment:
You should replace deleting the annotation and generating disable event to NodeStage. Or this event will appear each time when a related pod restarts